src/types/main.d.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * For default interfaces that are frequently used and extensions:
+ *   - HTMLElement: h5s and its inner objects.
+ *   - Extension of objects that are external to the library.
+*/
+
+interface opts {
```

Comment:
could you rename this one to `configuration`?  I know we currently use otps but I would like to change this as well. Thanks